### ISSUE TITLE
Support for commands with multiple parameters

### ISSUE THREAD

**Issue by bavovanachte** [bicarlsen/easy-scpi#4]
I'm trying to enter a command to my SCPI-enabled device that takes multiple parameters:

Concretely"

`MEMory:WORD <address>,<value>`

With a concrete invocation of:

`MEM:WORD #H0196, 511`

However, when I try to execute `inst.mem.word(0x196, 512)`, I get:

```
Traceback (most recent call last):
  File "easy_scpi_test.py", line 20, in <module>
    main()
  File "easy_scpi_test.py", line 13, in main
    inst.mem.word(0x196, 512)
TypeError: __call__() takes from 1 to 2 positional arguments but 3 were given
```

**Comment by bicarlsen** [comment]
Thank you for the suggestion. I just pushed a commit. Can you please pull the new code and test it out before I push it to PyPI?

**Comment by bicarlsen** [comment]
This should be resolved in [version 0.1.1](https://pypi.org/project/easy-scpi/0.1.1/).
